modules/stats.php

```diff
@@ -488,44 +626,79 @@ function stats_reports_page( $main_chart_only = false ) {
 		die;
 }
 
+/**
+ * Stats Convert Admin Urls.
+ *
+ * @access public
+ * @param mixed $html HTML.
+ * @return void
+ */
 function stats_convert_admin_urls( $html ) {
 	return str_replace( 'index.php?page=stats', 'admin.php?page=stats', $html );
 }
 
+/**
+ * Stats Convert Image URLs.
+ *
+ * @access public
+ * @param mixed $html HTML.
+ * @return void
+ */
 function stats_convert_image_urls( $html ) {
 	$url = set_url_scheme( 'https://' . STATS_DASHBOARD_SERVER );
 	$html = preg_replace( '|(["\'])(/i/stats.+)\\1|', '$1' . $url . '$2$1', $html );
 	return $html;
 }
 
+/**
+ * Stats Convert Chart URLs.
+ *
+ * @access public
+ * @param mixed $html HTML.
+ * @return void
+ */
 function stats_convert_chart_urls( $html ) {
 	$html = preg_replace_callback( '|https?://[-.a-z0-9]+/wp-includes/charts/([-.a-z0-9]+).php(\??)|',
-			create_function(
-				'$matches',
-				// If there is a query string, change the beginning '?' to a '&' so it fits into the middle of this query string
-				'return "admin.php?page=stats&noheader&chart=" . $matches[1] . str_replace( "?", "&", $matches[2] );'
-			),
-			$html );
+		create_function(
+			'$matches',
+			// If there is a query string, change the beginning '?' to a '&' so it fits into the middle of this query string.
+			'return "admin.php?page=stats&noheader&chart=" . $matches[1] . str_replace( "?", "&", $matches[2] );'
+		),
+		$html );
 	return $html;
 }
 
+/**
+ * Stats Convert Post Title HTML
+ *
+ * @access public
+ * @param mixed $html HTML.
+ * @return void
+ */
 function stats_convert_post_titles( $html ) {
 	global $wpdb, $stats_posts;
 	$pattern = "<span class='post-(\d+)-link'>.*?</span>";
 	if ( !preg_match_all( "!$pattern!", $html, $matches ) )
 		return $html;
 	$posts = get_posts( array(
-		'include' => implode( ',', $matches[1] ),
-		'post_type' => 'any',
-		'post_status' => 'any',
-		'numberposts' => -1,
-	));
+			'include' => implode( ',', $matches[1] ),
```

Comment:
Let's move the array declaration to a new line to make the double indentation look right? Same thing with the closing braces.